### PR TITLE
Fix  spaces

### PR DESCRIPTION
**Pull request by AtikSkywalker**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -9,25 +9,15 @@ const mongoose = require('mongoose');
 mongoose.connect('mongodb://localhost/contacts-db');
 const db = mongoose.connection
 
-db.on('error',(err)=>{
-
-
+db.on('error', (err) => {
     console.log(err)
```
Comment (tarekahsan709) [inline server.js:13]: you should always use any prefix before showing the log and use correct log type. For example this is a error log so you should use `console.error("Database Error:" err);`

File: server.js [hunk 2]
```diff
@@ -9,25 +9,15 @@ const mongoose = require('mongoose');
...
 
-
-
 )
 
-
 db.once('open', () => {
-
-
     console.log('Database connected Successfully')
```
Comment (tarekahsan709) [inline server.js:19]: the log type is okay but you should use a log prefix. For example console.log("Database: Connection has successful");

**Review by tarekahsan709 — COMMENTED** [review]
Please fix those review.

File: server.js (hunk 1 above)
Comment (AtikSkywalker) [reply server.js:13]: ok

File: server.js (hunk 2 above)
Comment (AtikSkywalker) [reply server.js:19]: ok

**Pull request merged**